### ISSUE TITLE
If possible, rewrite the rulesets in a way that does not require HTTPS->HTTPS rules.

### ISSUE THREAD

**Issue by koops76** [EFForg/https-everywhere#10843]
Related: #10839
- [ ] 7Headlines.com.xml
- [ ] Aeriagames.xml
- [ ] AJC.com.xml
- [ ] basekit.com.xml
- [ ] Blip.xml
- [ ] CA-State-Board-of-Equalization.xml
- [ ] Caltech.xml
- [ ] cambridgeshire.gov.uk.xml
- [ ] Care-Net.xml
- [ ] Comic-Con-Intl.xml
- [ ] Cru.org.xml
- [ ] DailyDot.xml
- [ ] Epic-Systems.xml
- [ ] Fasthosts.xml
- [ ] First_Class_Magazine.se.xml
- [ ] Google.xml
- [ ] HealthCheckUSA.xml
- [ ] InkFrog.com.xml
- [ ] Intelliworks_Chat.com.xml
- [ ] Khalsa-Credit-Union.xml
- [ ] KKH.de.xml
- [ ] Knappschaft-Bahn-See.xml
- [ ] MoveOn.xml
- [ ] MyEdAccount.Com.xml
- [ ] MYEDDEBT.com.xml
- [ ] National_Geographic.xml
- [ ] Necuhb.xml
- [ ] Numergy.xml
- [ ] Pair-Networks.xml
- [ ] Parkerpen.com.xml
- [ ] Perfect_Audience.xml
- [ ] Plymouth-University.xml
- [ ] Sberbank_of_Russia.xml
- [ ] SPajIT.xml
- [ ] spectator.co.uk.xml
- [ ] Telus.xml
- [ ] US-Dept-of-Veterans-Affairs.xml
- [ ] VEVO.xml
- [ ] Volgistics.xml
- [ ] Yimg.com.xml
- [ ] yWorks.com.xml
## Done
- [x] AdultShopping.com.xml (#11215)
- [x] Agol.dk.xml (#11210)
- [x] Austin_Fit_Magazine.com.xml (#11216)
- [x] BME.hu_incomplete.xml (#11217)
- [x] BME.hu.xml (#11217)
- [x] CafePress.xml (#10843)
- [x] Cashback.co.uk.xml (#11220)
- [x] CoolCart.Net.xml (#11024)
- [x] CuidadoDeSalud.gov.xml (#11024)
- [x] Dell.xml (#10829)
- [x] Deutsche-Rentenversicherung.de.xml (#11227)
- [x] Demonoid.xml (#11221)
- [x] EarthLink.xml (#10804)
- [x] ED.gov.xml (#11455)
- [x] Ghostery_metrics_optout.xml (#10849)
- [x] Kettering.gov.uk.xml (#11605)
- [x] Kiva.org.xml (#10232)
- [x] Linuxaria.com.xml (#11188)
- [x] Minijob-zentrale.de.xml (#11209)
- [x] Mutterschaftsgeld.de.xml (#11212)
- [x] Ntop.org.xml (#10061)
- [x] Sandbag.xml (#11243)
- [x] SourceFoundry.org.xml (#11336)
- [x] Stack-Exchange.xml (#10844)
- [x] Stallman.org.xml (#11213)
- [x] Unblocked.xml (#11208)
- [x] Unity.xml (#11338)
- [x] Virus_Bulletin.xml (#11228)

**Comment by cschanaj** [comment]
Noted that the following rulesets do not rewrite HTTPS.  
```
bfffe47dc9d0e3cc663ae55b4194150d7b2e7c996c753a16201aeece1ff526ae  Cato-Institute.xml
7a5412172a8a5c73ff51b177413b29539199397763dc5b11024ec0b7423df9c9  Epson.xml
f0da196f4b2f1f2a8fb0e7960c6e577b90781baf21e42dc677b041d13773f187  GoogleMaps.xml
b37bc0c3b107c55924fa5487231eab4004c6e6e6d581d7fac38ab2b1138c7d35  LiveJournal.xml
5f637cd0be002fb0f0cb2c3d13bc35d6baeef5ea76fccb62d93bf4d5b182bf9d  Tesco.xml
4a8a859f2bf60badb5d1f2363d4fb0c501ec63d8a5d11838e142afe913408375  Yandex.xml
667f0af26100508747674a3000fec0f25d4b73da00552894b743666bbba269c1  ZeniMax-Media.xml
```

**Comment by cschanaj** [comment]
Please also note the following rulesets found by the following command in https://github.com/EFForg/https-everywhere/pull/11024#pullrequestreview-49353302.
```bash
grep -Fl '^(?:http://(?:www\.)?|https://)' *.xml
```

```
fa3af743b1c53d079a9203e4876402e083fa4f5541b37aafbc916dc5c7c455e6  Care-Net.xml
961a6238a3fbb95a5f04b424843a8797185dd067488186944281f9b8c5314502  CA-State-Board-of-Equalization.xml
3bd8a37cd7bae417e479bfce375448bc0a557c7cd71dda1697a78fc250d4501c  EarthLink.xml
edbf455dd9b2344758f0e3861ebfe53c956596a2b71a36bb782cbb5a379bf527  HealthCheckUSA.xml
9ef73c2c7642cb622145154e9b86859f1b55598fd5a97f29a9bde51764528fe6  MyEdAccount.Com.xml
944ee1959b21eb47077b1bad494a3be3985c59f18b472b2ea0c9204c34ce5a2c  MYEDDEBT.com.xml
6579a09fbc23ca93ce9956cd0c8c250f5a007a687a836e4a0b62a919c5744eb7  Volgistics.xml
```

See also #10804. Thanks!!

**Comment by cschanaj** [comment]
Also note that the following ruleset
```
3d5bde747e8aaff058056b970800a53151f89006704db8cb2559e55f2ac8ef9b  ED.gov.xml
```
found by
```
grep -Fl '^(http://(www\.)?|https://)' *.xml
```

**Comment by Bisaloo** [comment]
https://github.com/EFForg/https-everywhere/pull/10087 updated `basekit.com` but the https->https rewrite can't be removed without breaking stuff.

**Comment by cschanaj** [comment]
@koops76 why do you open a new issue #11993 for this?

**Comment by koops76** [comment]
@cschanaj It's on 3rd page now and no one sees it, so I created a new issue for the rest of the domains.

**Comment by cschanaj** [comment]
@koops76 there are pages of issues listed behind the 3rd pages, I don't really agree this is a good idea.

**Comment by koops76** [comment]
@cschanaj No one notices it, that's why I re-opened this issue.

**Comment by cschanaj** [comment]
@koops76 imagine if everyone is doing this because nobody notice their issues, it will be pretty bad since bugs report and other issues will never get to the top.

**Comment by koops76** [comment]
@cschanaj It's quite an important issue, and it's not my personal issue but a project which benefits HTTPS Everywhere as whole. We need this to disable listening for HTTPS connections and improve performance.

**Comment by jeremyn** [comment]
@koops76 I agree with @cschanaj , please don't make duplicate issues. Aside from @cschanaj 's valid point of "what if everyone did this?", it also makes it harder to see the discussion around these problems like @Bisaloo 's and @cschanaj 's comments above.

It's best to consider these collection/checklist issues as being for your own reference, not for general viewing, in which case it doesn't matter how far down the queue it falls.

Please feel free to share any thoughts you have on making these issues more discoverable, if you want. Maybe adding certain GitHub labels would help?

**Comment by koops76** [comment]
@jeremyn Can you bump this issue up somehow?

**Comment by jeremyn** [comment]
I'm not sure how I would do that, if I wanted to. The project is almost entirely volunteer-driven except for a few paid EFF people who also have other responsibilities.

**Comment by ficofabrid** [comment]
Rewrite phoronix.com: https://www.phoronix.com/scan.php?page=news_item&px=HTTPS-Default-Everyone

**Comment by jeremyn** [comment]
@strugee What's going on with the bot?

**Comment by strugee** [comment]
@jeremyn yikes! That shouldn't be happening.

Did you (un)check a bunch of boxes? If so I bet GitHub sent a bunch of edit events to the bot when that happened and the bot complained due to the missing issue type metadata. In theory this shouldn't happen as of #13294 landing but I wrote that PR pretty late at night, and I think I forgot to test the date handling code.

If this sounds right to you I'll look into this and hopefully prepare a PR fixing it :thumbsup:

**Comment by strugee** [comment]
@jeremyn yikes! That shouldn't be happening.

Did you (un)check a bunch of boxes? If so I bet GitHub sent a bunch of edit events to the bot when that happened and the bot complained due to the missing issue type metadata. In theory this shouldn't happen as of #13294 landing but I wrote that PR pretty late at night, and I think I forgot to test the date handling code.

If this sounds right to you I'll look into this and hopefully prepare a PR fixing it :thumbsup:

**Comment by jeremyn** [comment]
@strugee This isn't my PR, I just noticed the bot having trouble. Maybe @koops76 could give more diagnostic information.

**Comment by koops76** [comment]
@strugee Each time I check the checkbox it sends an edit event, this is why the bot posted many times.

**Comment by jeremyn** [comment]
@strugee See also issue https://github.com/EFForg/https-everywhere/issues/3069 which has a similar problem.

**Comment by strugee** [comment]
Filed #13460 to fix this behavior.
